includes/SiteGen/SiteGen.php

```diff
@@ -823,11 +844,13 @@ function ( $key ) use ( $menu_patterns_slugs ) {
 						'hiivetoken' => HiiveConnection::get_auth_token(),
 						'prompt'     => array(
 							'site_description' => $site_description,
+							'keywords'   => wp_json_encode( $keywords ),
```

Comment:
I guess we can get rid of this now, we aren't using keywords anymore

reference:
https://github.com/newfold-labs/cf-worker-ai-sitegen/blob/main/src/routes.js#L48
https://github.com/newfold-labs/cf-worker-ai-sitegen/blob/main/src/controllers/ai.js#L19

Reply:
I think it's ok to send it as part of the the prompt and not its own property for now, just in case we want to use it later?

Reply:
Works, but whatever we send here will be propagated to gpt for content gen, it will be included as the part of prompt, so if we're okay with that, it's fine else we should remove.

Could be okay considering we have more tokens with 4o mini, that's the only caveat I see, we should also consider testing if that degrades / elevates the quality of generation

Reply:
Token-wise, with 4o-mini we're fine.

Most of the testing we've been doing have included keywords in the prompt without an obvious effect. I think the module should send it, and the service can take it out if needed since that side doesn't need a plugin release.

Reply:
I'm fine with that then :)